pywren/wrenhandler.py

```diff
@@ -46,7 +55,7 @@ def download_runtime_if_necessary(s3conn, runtime_s3_bucket, runtime_s3_key):
     """
 
     # get runtime etag
-    runtime_meta = s3conn.meta.client.head_object(Bucket=runtime_s3_bucket, 
+    runtime_meta = s3_client.head_object(Bucket=runtime_s3_bucket,
```

Comment:
It will be a good thing to think about what parts of wrenhandler cannot be handled by our existing storage API. I guess right now this includes `head_object`, `download_file` and `upload_file` ?